cmake/README.md

```diff
@@ -16,6 +16,7 @@ The following variables are provided for oneDPL configuration:
 | ONEDPL_USE_AOT_COMPILATION   | BOOL   | Enable ahead-of-time compilation for the GPU or CPU device types                              | OFF           |
 | ONEDPL_ENABLE_SIMD           | BOOL   | Enable SIMD vectorization by passing an OpenMP SIMD flag to the compiler if supported; the flag is passed to user project compilation string if enabled | ON           |
 | ONEDPL_AOT_ARCH              | STRING | Architecture options for ahead-of-time compilation, supported values can be found [here](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top/compilation/ahead-of-time-compilation.html)                                                                                            | "*" for GPU device and "avx" for CPU device |
+| TEST_EXPLICIT_KERNEL_NAMES   | STRING | Control usage of Kernel names in tests [AUTO, ALWAYS]: if AUTO then Kernel names are not used in tests when unnamed lambda option is switched on; if ALWAYS then Kernel names are used in tests explicitly | AUTO          |
```

Comment:
This may be a bit cleaner:
```suggestion
| TEST_EXPLICIT_KERNEL_NAMES   | STRING | Control kernel naming. Affects only oneDPL test targets. Supported values: AUTO, ALWAYS. AUTO: rely on the compiler if "Unnamed SYCL lambda kernels" feature is on, otherwise provide kernel names explicitly; ALWAYS: provide kernel names explicitly | AUTO          |
```

Reply:
This change were applied (committed)